pkg/hope/hypervisors/hypervisor.go

```diff
@@ -113,3 +125,30 @@ func GetEnginePlans(hypervisors []Hypervisor) ([]EngineBuildPlan, error) {
 
 	return retVal, nil
 }
+
+func HasNode(hv Hypervisor, node string) (bool, error) {
```

Comment:
These don't really do anything beyond offering a more semantic wrapper. I prefer this over client logic being needed to resolve these.